src/bidiMapper/domains/network/NetworkManager.ts

```diff
@@ -111,40 +100,40 @@ export class NetworkManager {
     cdpClient.on(
       'Network.loadingFailed',
       (params: Protocol.Network.LoadingFailedEvent) => {
-        networkProcessor
+        networkManager
           .#getOrCreateNetworkRequest(params.requestId)
           .onLoadingFailedEvent(params);
-        networkProcessor.#forgetRequest(params.requestId);
+        networkManager.#forgetRequest(params.requestId);
       }
     );
 
     cdpClient.on(
       'Network.loadingFinished',
       (params: Protocol.Network.RequestServedFromCacheEvent) => {
-        networkProcessor.#forgetRequest(params.requestId);
+        networkManager.#forgetRequest(params.requestId);
       }
     );
 
-    return networkProcessor;
+    return networkManager;
   }
 
   dispose() {
-    for (const request of this.#requestMap.values()) {
+    for (const request of this.#networkStorage.requestMap.values()) {
```

Comment:
There should be a method I on the networkStorage that does the dispose and here we should just call that.

Reply:
Fair. Done.